support_scripts/generate_json_rules.sh

```diff
@@ -0,0 +1,33 @@
+URL="https://docs.aws.amazon.com/config/latest/developerguide"
```

Comment:
Should have a shebang here
```suggestion
#!/usr/bin/env bash

URL="https://docs.aws.amazon.com/config/latest/developerguide"
```

Reply:
Yeah noticed that one a bit late. Cheers